client/src/components/CalcIcon.js

```diff
@@ -10,19 +10,56 @@ const CalcIcon = styled.button`
   color: ${(props) => calcColor[props.color].textcolor};
   background: none;
   font-family: 'Rubik', sans-serif;
+  box-shadow: 0 0 0.25em ${(props) => calcColor[props.color].shadowcolor};
 `;
 
 const calcColor = {
-  perfect: { bordercolor: '#ff0084', textcolor: '#ff0084' },
-  veryhigh: { bordercolor: '#ff00f2', textcolor: '#ff00f2' },
-  high: { bordercolor: '#ff9100', textcolor: '#ff9100' },
-  moderate: { bordercolor: '#FFD800', textcolor: '#FFD800' },
-  average: { bordercolor: '#59ff00', textcolor: '#59ff00' },
-  okay: { bordercolor: '#00ff6a', textcolor: '#00ff6a' },
-  substandard: { bordercolor: '#00ccff', textcolor: '#00ccff' },
-  low: { bordercolor: '#006AFF', textcolor: '#006AFF' },
-  verylow: { bordercolor: '#bb00ff', textcolor: '#bb00ff' },
-  almostnone: { bordercolor: '#7700ff', textcolor: '#7700ff' },
+  perfect: {
+    bordercolor: '#ff0084',
+    textcolor: '#ff0084',
+    shadowcolor: '#ff0084',
```

Comment:
are these colors always the same?
-> reduce it to a single property.

Besides that, this is a case for camelCase :).

Reply:
you are right, reduced this to the minimum :)